osu.Game.Rulesets.Catch/UI/CatcherArea.cs

```diff
@@ -45,20 +43,28 @@ public Catcher Catcher
         /// </summary>
         private int currentDirection;
 
+        // TODO: support replay rewind
+        private bool lastHyperDashState;
```

Comment:
Did the previous variant (trails in catcher) support rewind already? Maybe I'm just tired but I don't follow how one could work but not the other.

Reply:
Neither previous the previous variant in this PR nor the code before this PR support rewind properly.
I added a TODO as a remainder, as it is now more obvious the code is not handling rewinding.